testdata/project-v3-multigroup/config/default/kustomization.yaml

```diff
@@ -25,11 +25,15 @@ bases:
 #- ../prometheus
 
 patchesStrategicMerge:
-  # Protect the /metrics endpoint by putting it behind auth.
-  # If you want your controller-manager to expose the /metrics
-  # endpoint w/o any authn/z, please comment the following line.
+# Protect the /metrics endpoint by putting it behind auth.
+# If you want your controller-manager to expose the /metrics
+# endpoint w/o any authn/z, please comment the following line.
 - manager_auth_proxy_patch.yaml
 
+# Mount the controller config file for loading manager configurations
+# through a ComponentConfig type
+#- manager_config_patch.yaml
+
```

Comment:
WDYT about we use this config in the e2e tests to ensure this feature? See: https://github.com/kubernetes-sigs/kubebuilder/blob/master/test/e2e/v3/generate_test.go#L79

Reply:
Yeah, I can get an e2e test for this as well.

Reply:
Just to confirm we're taking about uncommenting this line but not using passing the `--component-config` flag into the e2e `init` command, right? The idea is to test if we can uncomment and use kustomize without breakages, not to test the controller booting while loading from the file, right?

Reply:
For now I've added https://github.com/kubernetes-sigs/kubebuilder/pull/1790/files#diff-937f62ec43b0cd36407ae8e72d9f605f24f53968f898d3a4ada2ba47c4c70455R184-R186 which will handle at least testing that kustomize doesn't break when this is uncommented and that they controller still boots.

Reply:
solved